src/main/java/com/yegor256/xsline/StSchema.java

```diff
@@ -90,14 +95,15 @@ public StSchema(final XML schema) {
 
     /**
      * Validate it.
+     *
      * @param schema The schema
      * @param xml The XML
      * @return The same XML
      */
     private static XML validate(final XML schema, final XML xml) {
         final Collection<SAXParseException> violations;
-        if (schema == null) {
-            violations = xml.validate();
```

Comment:
@volodya-lombrozo try jcabi-xml 0.33.4, it has public class `ClasspathResolver`, an instance of which you can pass to `xml.validate()` when schema is not provided (`schema` is `null`)

Reply:
@yegor256 I'm getting this:

> Could not find artifact com.jcabi:jcabi-xml:jar:0.33.4 in central

Are you sure it's released?